### PR TITLE
Initial CCPA Support

### PR DESCRIPTION
**Pull request by pm-shashank-jain**

### CODE REVIEW

File: src_new/constants.js [hunk 1]
```diff
@@ -70,7 +70,12 @@ exports.CONFIG = {
 	"DEFAULT_USER_ID_MODULE": "0",
 	"DEFAULT_IDENTITY_ONLY": "0",
 	"DEFAULT_GDPR_CONSENT": "0",
-	"DISABLE_AJAX_TIMEOUT": "disableAjaxTimeout"
+	"DISABLE_AJAX_TIMEOUT": "disableAjaxTimeout",
+	"CCPA_CONSENT": "ccpa",
+	"CCPA_CMPAPI": "ccpaCmpApi",
+	"CCPA_TIMEOUT": "ccpaTimeout",
+	"DEFAULT_CCPA_CMPAPI": "iab",
+	"DEFAULT_CCPA_TIMEOUT": 10000,
```
Comment (pm-harshad-mane) [inline src_new/constants.js]: default timeout looks too long, is it same for GDPR?

File: src_new/adapters/prebid.js [hunk 2]
```diff
@@ -617,12 +617,18 @@ function fetchBids(activeSlots, impressionID){
 				};
 
 				if (CONFIG.getGdpr()) {
-					prebidConfig["consentManagement"] = {
+					prebidConfig["consentManagement"]["gdpr"] = {
```
Comment (pm-harshad-mane) [inline src_new/adapters/prebid.js]: prebidConfig["consentManagement"] is not assigned a value as an empty object.
isn't it throwing some error?

File: src_new/constants.js (hunk 1 above)
Comment (pm-shashank-jain) [reply src_new/constants.js]: yes harshad
also prebid has the same default timeout of  10000 
ref-> http://prebid.org/dev-docs/modules/consentManagementUsp.html

File: src_new/adapters/prebid.js (hunk 2 above)
Comment (pm-shashank-jain) [reply src_new/adapters/prebid.js]: Fixed the issue. Thanks :)

**Comment by pm-shashank-jain** [comment]
Created a new PR with staged_nightly #150

**Pull request closed without merging**